### PR TITLE
add resultset throw exception

### PR DESCRIPTION
**Pull request by datian9966**
such as title

### CODE REVIEW

File: client/src/main/java/com/vesoft/nebula/client/graph/ResultSet.java [hunk 1]
```diff
@@ -36,7 +36,11 @@ public ColumnValue get(int index) {
         }
 
         public ColumnValue get(String key) {
-            return this.row.columns.get(columns.indexOf(key));
+            int index = columns.indexOf(key);
+            if ( index == -1 ){
+                throw new RuntimeException("Cannot get field because the key '"+key+"' is not exist");
```
Comment (darionyaphet) [inline client/src/main/java/com/vesoft/nebula/client/graph/ResultSet.java:41]: this should be Illegal Argument Exception ?

File: client/src/main/java/com/vesoft/nebula/client/graph/ResultSet.java [hunk 2]
```diff
@@ -36,7 +36,11 @@ public ColumnValue get(int index) {
         }
 
         public ColumnValue get(String key) {
-            return this.row.columns.get(columns.indexOf(key));
+            int index = columns.indexOf(key);
+            if ( index == -1 ){
```
Comment (darionyaphet) [inline client/src/main/java/com/vesoft/nebula/client/graph/ResultSet.java:40]: extra space

**Comment by darionyaphet** [comment]
Seems some thing building failed , you should make sure it could pass the `maven compile package`

**Comment by datian9966** [comment]
@darionyaphet  OK~

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**